### ISSUE TITLE
The build process doesn't work!

### ISSUE THREAD

**Issue by mjbom** [kartoza/docker-mapserver#4]
The command '/bin/sh -c chmod o+x /usr/local/bin/mapserv' returned a non-zero code: 1

**Comment by NyakudyaA** [comment]
@mjbom I will look at it when I am free

**Comment by hkotkanen** [comment]
`mapserv` binary isn't there because the build process fails. Problem for me was that `protobuf-c` library can't be found on the system, but I was able to move forward by following the compiler's hint: by adding `-DWITH_PROTOBUFC=0` to CMake flags in [setup.sh](https://github.com/kartoza/docker-mapserver/blob/4141c485f97a520d8c5f3147a808072757e2a900/setup.sh#L45). This dependency is [apparently](https://mapserver.org/development/rfc/ms-rfc-119.html) needed for Mapbox Vector Tile support which I didn't need.

After writing this, I noticed that this is exactly what PR #5 is doing.
